File: src/main/java/FizzBuzz.java
public class FizzBuzz {

  public static String fizzBuzz(int n) {
    //BEGIN_CHALLENGE

    String result;
    if (n % 15 == 0) {
      result = "FizzBuzz";
    }
    else if (n % 5 == 0) {
      result = "Buzz";
    }
    else if (n % 3 ==0) {
      result = "Fizz";
    }
    else result = Integer.toString(n);
    return result;

    //END_CHALLENGE
  }
}

File: app/fizzbuzz.js
function fizzbuzz(data) {
  //Solve the challenge here and pass all the tests
  //BEGIN_CHALLENGE
  var result;
  if(data % 15 == 0) { result = "FizzBuzz"; }
  else if(data % 5 == 0) {result = "Buzz"; }
  else if(data %3 ==0 ) {result = "Fizz"; }
  else result = data.toString();
  return result;
  //END_CHALLENGE

}

module.exports = fizzbuzz;
